app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Typography, Avatar, Dropdown } from "neetoui";
+
+const { Menu, MenuItem } = Dropdown;
+
+export const getContactsTableColumnData = (
```

Comment:
`build` is a better keyword that `get` over here. `get` should be the last resort when naming util functions. `get` usually suggests that all the data that we need is already present in its final structure/state. We just need to get it as it's. But here, we are building the data based on the arguments that we've passed.